release/cc/create_artifacts.py

```diff
@@ -0,0 +1,184 @@
+# Copyright 2016 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Module to create cc toolchain artifacts including targets and METADATA."""
```

Comment:
does this create a new file or add the targets to an existing file
if the latter
Module to add cc toolchain targets to the BUILD files (for all configs) and METADATA (or something like that?)

Reply:
Changed to Module to add cc toolchain targets to BUILD files and METADATA.